### PR TITLE
Example of ignition template

### PR DESCRIPTION
**Pull request by dahaic**
This is just an example how ignition template can be done.

### CODE REVIEW

**Comment by jan-cerny** [comment]
@dahaic The template for disabled services has been already introduced by #5729. Please remove it from you PR.

File: ssg/templates.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def service_disabled(data, lang):
     return data
 
 
-@template(["ansible", "bash", "oval", "puppet"])
+@template(["ansible", "bash", "oval", "puppet", "ignition"])
```
Comment (jan-cerny) [inline ssg/templates.py:300]: Don't forget to add it to the Developer guide.

File: shared/templates/template_IGNITION_service_enabled [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+      - name: {{{ DAEMONNAME }}}.service
+        enabled: true
```
Comment (JAORMX) [inline shared/templates/template_IGNITION_service_enabled:11]: could we add a template for kubernetes as well?

**Comment by jan-cerny** [comment]
@dahaic What are your plans?

**Comment by jan-cerny** [comment]
@dahaic bump

**Comment by matejak** [comment]
I believe that this functionality has been implemented by #5729, and I guess that this PR served as an inspiration. Thanks @dahaic!

**Pull request closed without merging**